tests/test_database.py

```diff
@@ -4,31 +4,17 @@
 import database_connector
 import database_manager
 
-
-def create_test_database(database_connector):
-    database_connector.execute("CREATE DATABASE test_db")
-
-
-def set_test_database(database_connector):
-    database_connector.set_database("test_db")
-
-def set_database_to_none(database_connector):
-    database_connector.set_database(None)
-
-
-def delete_test_database(database_connector):
-    database_connector.execute("DROP DATABASE test_db")
-
-
-def create_items_test_table(database_connector):
+def create_test_database_setup(database_connector):
+    database_connector.execute("CREATE DATABASE TestDB") #create database
+    database_connector.set_database("TestDB") #set database
     database_connector.execute(
-        "CREATE TABLE items (item_id VARCHAR(45) NOT NULL PRIMARY KEY, vector_path VARCHAR(255) NOT NULL, png_path VARCHAR(255) NOT NULL);")
-
-
-def create_tags_test_table(database_connector):
+        "CREATE TABLE Items (ItemId VARCHAR(45) NOT NULL PRIMARY KEY, VectorPath VARCHAR(255) NOT NULL, PngPath VARCHAR(255) NOT NULL);") #create items table
     database_connector.execute(
-        "CREATE TABLE tags (tag_id VARCHAR(45) NOT NULL PRIMARY KEY, tag VARCHAR(255) CHARACTER SET UTF8MB4 COLLATE utf8mb4_unicode_ci NOT NULL,item_id VARCHAR(45) NOT NULL);")
+        "CREATE TABLE Tags (TagId VARCHAR(45) NOT NULL PRIMARY KEY, Tag VARCHAR(255) CHARACTER SET UTF8MB4 COLLATE utf8mb4_unicode_ci NOT NULL,ItemId VARCHAR(45) NOT NULL);") #create tags table
```

Comment:
1. Nit: space after comma
2. You can split query over multiple lines, e.g: https://stackoverflow.com/a/10660477

Reply:
fixed